pos_payment_terminal/__manifest__.py

```diff
@@ -1,14 +1,14 @@
-# -*- coding: utf-8 -*-
 # © 2014-2016 Aurélien DUMAINE
 # © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# © 2019 Akretion (Pierrick Brun <pierrick.brun@akretion.com>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'POS Payment Terminal',
-    'version': '10.0.0.1.1',
+    'version': '12.0.0.1.0',
     'category': 'Point Of Sale',
     'summary': 'Manage Payment Terminal device from POS front end',
-    'author': "Aurélien DUMAINE,Akretion,Odoo Community Association (OCA)",
+    'author': "Akretion, Odoo Community Association (OCA)",
```

Comment:
why did you removed the author ?

Reply:
I've left only the organisations in this field, isn't it how it is supposed to work ? Aurélien and Alexis are on the top of the file